encryptme_stats/metrics.py

```diff
@@ -583,14 +605,18 @@ def vpn_session():
 
     :return: list of dictionaries with connections statistics
     """
-    empty = []
-    return empty  # disable vpn_session metrcis for now
+    try:
+        empty = []
+        return empty  # disable vpn_session metrcis for now
```

Comment:
please enable sending stats and remove return here

Reply:
Done